syntax = "proto3";

package gbf;

import "domain.proto";

message RaidTweetResponse {
  string bossName = 1;
  string raidId = 2;
  string screenName = 3;
  int64 tweetId = 4;
  string profileImage = 5;
  string text = 6;
  int64 createdAt = 7;
  Language language = 8;
};
